build/media_source/system/js/inlinehelp.es6.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * @copyright  (C) 2021 Open Source Matters, Inc. <https://www.joomla.org>
+ * @license    GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+/**
+ * Toggles the display of inline help DIVs
+ *
+ * @param {String} toggleClass The class name of the DIVs to toggle display for
+ */
+Joomla.toggleInlineHelp = (toggleClass) => {
+  document.querySelectorAll(`div.${toggleClass}`)
+    .forEach((elDiv) => {
+      elDiv.classList.toggle('d-none');
+    });
+};
+
+document.querySelectorAll('.button-inlinehelp')
```

Comment:
```suggestion
[].slice.call(document.querySelectorAll('.button-inlinehelp'))
```

The reason is that `HTMLElement..querySelectorAll()` can return either `undefined` or a `node list`. Wrapping it with `[].slice.call()` we ensure that the returned value is ALWAYS an array so the `forEach` will never throw. As you have it right now if for any reason there aren't any elements with this class in the DOM the script will throw...

Reply:
Thank you for the tip!

Reply:
Np, you need to do the same @ line 13

Reply:
Already on it :)